frontends/concrete-python/concrete/fhe/mlir/context.py

```diff
@@ -2548,9 +2548,6 @@ def identity(self, resulting_type: ConversionType, x: Conversion) -> Conversion:
             and x.is_signed == resulting_type.is_signed
         )
 
-        if resulting_type.bit_width == x.bit_width:
```

Comment:
if it's a mistake we did in the past, shouldn't there be a test to avoid we do it again?

Reply:
yes that could  be a composition test relying on fhe.identity.

Reply:
I have added a test and fhe.refresh so fhe.identity stays as before.